meta-mender-core/recipes-mender/mender-shell/mender-shell.bb

```diff
@@ -32,6 +35,28 @@ do_compile() {
         V=1
 }
 
+python do_prepare_mender_shell_conf() {
+    import json
+
+    # If a mender-shell.conf has been provided in SRC_URI, use this one instead
+    src_conf = os.path.join(d.getVar("WORKDIR"), "mender-shell.conf")
+    if os.path.exists(src_conf):
+        bb.debug(1, "mender-shell.conf already present in ${WORKDIR}, return.")
+
+    mender_shell_conf = {}
```

Comment:
Should this be populated with a `json.load` from the original file, if there is one?

Update: See my comment below as well.